lib/src/js/live_chat.js

```diff
@@ -0,0 +1,84 @@
+function startLiveChat(license, userName, email) {
+    window.__lc = window.__lc || {};
+    window.__lc.license = license;;
+    window.__lc.asyncInit = true;
+    (function(n, t, c) {
+        function i(n) {
+            return e._h ? e._h.apply(null, n) : e._q.push(n)
+        }
+        var e = {
+            _q: [],
+            _h: null,
+            _v: "2.0",
+            on: function() {
+                i(["on", c.call(arguments)])
+            },
+            once: function() {
+                i(["once", c.call(arguments)])
+            },
+            off: function() {
+                i(["off", c.call(arguments)])
+            },
+            get: function() {
+                if (!e._h) throw new Error("[LiveChatWidget] You can't use getters before load.");
+                return i(["get", c.call(arguments)])
+            },
+            call: function() {
+                i(["call", c.call(arguments)])
```

Comment:
would you also need `if (!e._h) throw new Error("[LiveChatWidget] You can't use caller before load.");`?